vllm-speculative-decoding/config.yaml

```diff
@@ -0,0 +1,23 @@
+base_image:
+  image: nvcr.io/nvidia/pytorch:23.11-py3
+  python_executable_path: /usr/bin/python3
```

Comment:
just cuious why we need this base image? Can you add a cooment?

Reply:
Without this base image the build does not succeed. The baseten base image does not have `nvcc`, which is required for the developer build of vLLM.